Examples/OneSignalDemo/app/build.gradle

```diff
@@ -3,6 +3,14 @@ apply plugin: 'com.onesignal.androidsdk.onesignal-gradle-plugin'
 apply plugin: 'com.android.application'
 
 android {
+    signingConfigs {
+        debug {
+            storeFile file('/Users/elliotmawby/.android/debug.keystore')
```

Comment:
I believe Android Studio uses all the these as defaults automatically so we don't need to add these here.

Reply:
yeah, but If @emawby is building the apk from console it might be asking him to have this configured on the file.